<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>846</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>поставьте 4 автоматом пожалуйста</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>780</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbCoeff">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>780</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Task 1</string>
     </attribute>
     <widget class="QCustomPlot" name="graphLinInc" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>490</width>
        <height>330</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphLinDec" native="true">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>30</y>
        <width>490</width>
        <height>330</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphExpInc" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>490</width>
        <height>330</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphExpDec" native="true">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>400</y>
        <width>250</width>
        <height>165</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphCombinated" native="true">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>566</y>
        <width>250</width>
        <height>165</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>10</y>
        <width>120</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Линейный восходящий</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>696</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Линейный нисходящий</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>380</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Экспоненциальный восходящий</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>380</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Экспоненциальный нисходящий</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>813</x>
        <y>550</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Комбинация ЛВ, ЛН, ЭВ</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Task 2</string>
     </attribute>
     <widget class="QCustomPlot" name="graphEmbedRandom" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>490</width>
        <height>400</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphMyRandom" native="true">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>170</y>
        <width>500</width>
        <height>400</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>206</x>
        <y>150</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Встроенный рандом</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>725</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&quot;Мой&quot; рандом</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Task 3</string>
     </attribute>
     <widget class="QCustomPlot" name="graphTaskThree" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>680</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>170</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Минимум</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>200</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Максимум</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>230</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Мат. ожидание</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>260</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Дисперсия</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>290</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Стандартное отклонение</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>320</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ср. кв. отклонение</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>350</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ассиметрия</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>380</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Коэфициент ассиметрии</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>410</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Эксцесс</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>440</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Куртозис</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDispersionValue">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>260</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Дисперсия</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMeanValue">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>230</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Мат. ожидание</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelAssymetryValue">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>350</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ассиметрия</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelExcessValue">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>410</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Эксцесс</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSqrtDevValue">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>320</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ср. кв. отклонение</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMinimumValue">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>170</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Минимум</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelStdDevValue">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>290</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Стандартное отклонение</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelAssymetryCoeffValue">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>380</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Коэфициент ассиметрии</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMaximumValue">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>200</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Максимум</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCurtosisValue">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>440</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Куртозис</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>470</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Стационарность</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelIsStationarity">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>470</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Стационарность</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>316</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&quot;Мой&quot; рандом</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Task 4</string>
     </attribute>
     <widget class="QCustomPlot" name="graphTaskFourRandom1" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>500</width>
        <height>250</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphCovar" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>500</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphAutocovar" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>300</y>
        <width>500</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Встроенный рандом 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>280</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ковариация</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>510</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Плотность распределения</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>280</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Автоковариация</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>0</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Встроенный рандом 2</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphTaskFourRandom2" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>500</width>
        <height>250</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widgetTest" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>990</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Task 5</string>
     </attribute>
     <widget class="QCustomPlot" name="graphTaskFiveSinMod" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>990</width>
        <height>250</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_11">
     <attribute name="title">
      <string>Task 6</string>
     </attribute>
     <widget class="QCustomPlot" name="graphTaskSixOrig1" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>490</width>
        <height>230</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphTaskSixShifted1" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>490</width>
        <height>230</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphTaskSixOrig2" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>490</width>
        <height>230</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphTaskSixShifted2" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>260</y>
        <width>490</width>
        <height>230</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphTaskSixSpiked" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>490</width>
        <height>230</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_12">
     <attribute name="title">
      <string>Task 7</string>
     </attribute>
     <widget class="QCustomPlot" name="graphTaskSevenShifted" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>490</width>
        <height>230</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphTaskSevenAntiShifted" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>490</width>
        <height>230</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphTaskSevenSpiked" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>490</width>
        <height>230</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graphTaskSevenAntiSpiked" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>260</y>
        <width>490</width>
        <height>230</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Страница</string>
     </attribute>
     <widget class="QCustomPlot" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>301</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
